assets/RNAseq_report.Rmd

```diff
@@ -894,20 +924,23 @@ The original plot can be downloaded [here](./differential_gene_expression/plots/
 
 ```{r PCA_ifNoBatch_plot, echo=F, message=F, warning=F, out.width="80%", dpi=1200, fig.align='center'}
 ############################ PCA PLOTS ########################
-pcaData <- plotPCA(if (params$rlog) rld else vsd,intgroup=c("combfactor"),ntop = dim(if (params$rlog) rld else vsd)[1], returnData=TRUE)
+pcaData <- plotPCA(if (params$rlog) rld else vsd,intgroup=c("combfactor"), ntop=dim(if (params$rlog) rld else vsd)[1], returnData=TRUE)
 percentVar <- round(100*attr(pcaData, "percentVar"))
-pca <- ggplot(pcaData, aes(PC1, PC2, color=combfactor)) +
-    geom_point(size=3) +
-    xlab(paste0("PC1: ",percentVar[1],"% variance")) +
-    ylab(paste0("PC2: ",percentVar[2], "% variance")) +
-    theme(legend.title = element_blank()) +
-    coord_fixed() +
-    theme_bw() +
-    theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank())
+pca <- ggplot(pcaData, aes(PC1, PC2, color=combfactor, text=paste("PC1: ", PC1, "<br>", "PC2: ", PC2))) +
+            geom_point(size=3) +
+            xlab(paste0("PC1: ",percentVar[1],"% variance")) +
+            ylab(paste0("PC2: ",percentVar[2], "% variance")) +
+            coord_fixed() +
+            theme_bw() +
+            theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank()) +
+            guides(color=guide_legend(title=NULL))
 ggsave(plot = pca, filename = "differential_gene_expression/plots/PCA_plot.pdf", device = "pdf", dpi = 300)
 ggsave(plot = pca, filename = "differential_gene_expression/plots/PCA_plot.svg", device = "svg", dpi = 150)
 ggsave(plot = pca, filename = "differential_gene_expression/plots/PCA_plot.png", device = "png", dpi = 150)
-pca
+pca <- ggplotly(pca, tooltip = "text", width=750, height=400)
```

Comment:
Does it now already include `SampleName`?

Reply:
ah yes, I added that